docs/examples/promise/02_listen_to_blocks/README.md

```diff
@@ -4,6 +4,6 @@ This example shows how to subscribe to new blocks.
 
 It displays the block number every time a new block is seen by the node you are connected to.
 
-NOTE: The example runs until you stop it with CTRL+C
+NOTE: In this example we're calling the built-in unsubscribe() function after a timeOut of 20s.
```

Comment:
Rather do this example elsewhere - we want to provide an explore-like experience, i.e. run forever

Reply:
I added two separate examples as 04_unsubscribe